.github/ISSUE_TEMPLATE/language-service.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Language Service
+about: 'Issues pertaining to IntelliSense, autocomplete, code editing, etc. '
+
+---
+
+Type: LanguageService
+**----- Input information below -----**
+
+**Please review existing issues and our documentation at https://github.com/Microsoft/vscode-cpptools/tree/master/Documentation prior to filing an issue.**
+
+**Describe the bug**
+- OS and Version:
+- VS Code Version:
+- C/C++ Extension Version:
+- Other extensions you installed (and if the issue persists after disabling them):
+- A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+*Provide a code sample including configuration files such as c_cpp_properties.json*
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Additional context**
+*Call Stacks: For bugs like crashes, deadlocks, infinite loops, etc. that we are not able to repro and for which the call stack may be useful, please attach a debugger and/or create a dmp and provide the call stacks. Starting with 0.17.3, Windows binaries have symbols available in VS Code by setting your "symbolSearchPath" to "http://msdl.microsoft.com/download/symbols".*
+Add any other context about the problem here including log messages in your Output window ("C_Cpp.loggingLevel": "Information" in settings.json).
```

Comment:
May want to list "Debug" instead of "Information" for the logging level

Reply:
I pulled it out of the current template. Is it supposed to be "debug" ?